src/Propel/Generator/Platform/MysqlPlatform.php

```diff
@@ -80,17 +84,49 @@ protected function initialize(): void
     public function setGeneratorConfig(GeneratorConfigInterface $generatorConfig): void
     {
         parent::setGeneratorConfig($generatorConfig);
-        if ($defaultTableEngine = $generatorConfig->get()['database']['adapters']['mysql']['tableType']) {
+
+        $mysqlConfig = $generatorConfig->get()['database']['adapters']['mysql'];
+
+        if ($defaultTableEngine = $mysqlConfig['tableType']) {
             $this->defaultTableEngine = $defaultTableEngine;
         }
-        if ($tableEngineKeyword = $generatorConfig->get()['database']['adapters']['mysql']['tableEngineKeyword']) {
+        if ($tableEngineKeyword = $mysqlConfig['tableEngineKeyword']) {
             $this->tableEngineKeyword = $tableEngineKeyword;
         }
+        if ($uuidColumnType = $mysqlConfig['uuidColumnType']) {
+            $enable = strtolower($uuidColumnType) === 'native';
```

Comment:
[Minor] Wouldn't be better to have a standalone platform for MariaDb that extends a MySQL one?

Reply:
Yes, that would be great, but that is its own PR